Dockerfile

```diff
@@ -0,0 +1,55 @@
+# syntax=docker/dockerfile:1
+
+# GCC Base
+FROM gcc:11.2.0 as build
```

Comment:
I'd definitely like to see a multi stage build for a build image and then a runtime image where you copy over the binaries from the build image to the runtime image after you're done building. GCC is a massive container and its binaries are no longer needed when running DLU.

For ref: https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Great idea. I left it there for debugging, but I’ll clean it up and move everything to a small container

Reply:
Let me try an alpine container and see if I can get it working in that. Theoretically, it _should_ have all necessary runtime dependencies, but it might require installing some extra stuff related to libstdc++. I'm unsure though.